.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-20.04']
-        tox-env: [py38, dask]
+        tox-env: [py38, dask, mypy]
```

Comment:
Will it be enough to run mypy on 3.8...?

Reply:
Indeed, we will probably want to add more in the future?